Swift Vocabulary/Swift Vocabulary/View Controllers/WordsTableViewController.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  WordsTableViewController.swift
+//  Swift Vocabulary
+//
+//  Created by James Sedlacek on 5/19/20.
+//  Copyright © 2020 James Sedlacek. All rights reserved.
+//
+
+import UIKit
+
+class WordsTableViewController: UITableViewController {
+
+    var words: [VocabularyWord] = [VocabularyWord(word: "Variable", definition: "Variable Definition"),
+    VocabularyWord(word: "Constant", definition: "Constant Definition"),
+    VocabularyWord(word: "Function", definition: "Function Definition")
```

Comment:
Some longer definitions would be nice, and would expose how real examples would look in your app, exposing any problems with layout or user interface that might arise.